index.html

```diff
@@ -38,6 +39,204 @@ <h2 id="job-title">I am a Software Developer</h2>
                     <li><a href=""><img class="icon-interactions" src="images/Vector.png" alt="M-sign"></li></a>
                 </ul> 
             </section>
+            <section class="work">
+                <div>
+                    <div class="recent-work">
+                        <h2 id="work-intro">My Recent Work</h2>
+                        <a id="yoga-image" href=""><img src="images/Img Placeholder-1.png" alt="yoga girl"></a>
+                    </div>
+                    <div class="multi-post-div">
+                        <h2 id="multi-post-title" >Multi-Post Stories</h2>
+                        <div id="work-description">
+                            <p id="work-font">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                        </div>
+                        <div class="button-div">
+                            <ul class="button-list">
+                                <li><a href=""><button id="multi-post-button">css</button></a></li>
+                                <li><a href=""><button id="multi-post-button">html</button></a></li>
+                                <li><a href=""><button id="multi-post-button">bootstrap</button></a></li>
+                                <li><a href=""><button id="multi-post-button">Ruby</button></a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <a href=""><button id="first-see-project-button">See Project</button></a>
+                </div>
+                <div class="project-1">
+                    <a id="project-backdrop" href=""><img src="images/Img Placeholder-2.png" alt=""></a>
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image